_posts/2020-10-16-kaitai-struct-v0.9-released.md

```diff
@@ -99,12 +102,17 @@ fixes a lot of bugs and includes quite a few infrastructure improvements.
     Gradle plugin](https://github.com/valery1707/kaitai-gradle-plugin)
 * Infrastructure updates:
   * Unstable binary builds are available for all platforms after every CI build at Bintray ([#63](https://github.com/kaitai-io/kaitai_struct/issues/63))
-  * KSY language reference replaced with [documentation](https://doc.kaitai.io/ksy_diagram.html) generated from JSON schema
+  * KS language reference replaced with [documentation](https://doc.kaitai.io/ksy_diagram.html) generated from [the JSONSchema](https://github.com/kaitai-io/ksy_schema)
   * [https://formats.kaitai.io/](https://formats.kaitai.io/) is rebuilt automatically with CI/CD
   * Brand new modular CI/CD system for compiler, underlying
     CI-agnostic, working on multiple different OSes in parallel
     (Linux, Windows, macOS) and showing status at [https://ci.kaitai.io/](https://ci.kaitai.io/)
   * Generate test assertion specs from language-agnostic [KST specs](https://doc.kaitai.io/kst.html)
+  * Created [a list of awes😎me projects related to Kaitai Struct](https://github.com/kaitai-io/awesome-kaitai)
```

Comment:
A **related project**, and please don't put the emoji there (it's disturbing, and search engines won't like that):

```suggestion
  * Created [a list of awesome projects related to Kaitai Struct](https://github.com/kaitai-io/awesome-kaitai)
```

Reply:
The emoji was to convey message that the word awesome has some non-trivial meaning. Replaced with the image with awesome badge with alt text having the word "awesome".

Reply:
> The emoji was to convey message that the word awesome has some non-trivial meaning. Replaced with the image with awesome badge with alt text having the word "awesome".

I don't think that's necessary, and that any creative idea will actually help that. People that have ever heard about awesome lists will probably understand "a list of awesome projects", and those who have never heard about it will not understand the meaning of 😎, 🕶️ or whatever logo or symbol.

What about just "`awesome-kaitai` - list of projects related to Kaitai Struct"? Anyway, the name Awesome Kaitai would should be contained in the point.